src/python/ensembl/production/datafile/file_parsers.py

```diff
@@ -0,0 +1,336 @@
+#  See the NOTICE file distributed with this work for additional information
```

Comment:
This file would earn to be split into dedicated submodules, imho. Getting a bit long.

Reply:
See refactored layout